CHANGELOG.md

```diff
@@ -21,6 +19,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Fixed
 - Fixed CEDS `HEMCO_Config.rc` entries to emit TMB into the TMB species (and not HCOOH)
 - Added C6H14 emissions into the ALK6 species for CMIP6 & HTAPv3 inventories
+- Fixed typo: `KPP_INTEGERATOR_AUTOREDUCE` -> `KPP_INTEGRATOR_AUTOREDUCE`
```

Comment:
I don't think this update is needed if it's fixing something introduced in dev/14.6.0

Reply:
Now fixed in 9dcb384